js/nixos-site.js

```diff
@@ -1,3 +1,25 @@
+// FisherYates implementation
+$.fn.shuffleChildren = function(seedParam = Date.now()) {
+  var randomish = function(s) {
+    s = Math.sin(s) * 10000;
+    return s - Math.floor(s);
+  };
+
+  $.each(this.get(), function(index, el) {
+    var $el = $(el);
+    var $find = $el.children();
+    var len = $find.length;
+    var seed = randomish(seedParam);
+
+    while (--len) {
+      let randIndex = Math.floor(seed * (len + 1));
+      [$find[randIndex], $find[len]] = [$find[len], $find[randIndex]];
+    }
+
+    $el.empty().append($find);
+  });
+};
```

Comment:
(Repeating/rephrasing my comments from the previous pull request:)

For the Fisher-Yates shuffle to work, a new random number has to be used
each time an index is picked for swapping elements.

Also, the sinful function (used here to work around the lack of seedable PRNG
in the web browser) isn't fair.

How about dropping the hourly seed thing and just use `Math.random()` in the
`while` loop?

Reply:
This: https://github.com/NixOS/nixos-homepage/pull/1052#discussion_r1193377878

Reply:
I support this and I think that we should drop the hourly seed thingie.

Reply:
@drupol can you update this on your branch and reopen the original PR? Then we can merge something at all. I will hit the button if you ping me, let's fix forward at the worst.

Reply:
I created a new PR @ https://github.com/NixOS/nixos-homepage/pull/1081